### ISSUE TITLE
Some notion articles output no body data

### ISSUE THREAD

**Issue by alez156** [mchlkucera/notion-to-html#6]
I noticed that a lot of the notion cards I've been running through the Make automation result in no body data when they get into Webflow. It only happens with a some of the articles. 

Here's all the articles that run into this issue:
• https://reapertips.notion.site/6-things-I-wish-I-knew-when-I-started-using-REAPER-fe9df20906a24905b635fec133e6cc14
• https://reapertips.notion.site/This-card-also-doesnt-work-e7b27828debd47f6abba16cf734bd8dc
• https://reapertips.notion.site/Quickest-way-to-sidechain-in-REAPER-e4a58c58218b49dab5193bb5d4bfe075
• https://reapertips.notion.site/Capture-anything-in-REAPER-with-Global-Sampler-51e7d2f159db49c8b79d400a8db1b7da
• https://reapertips.notion.site/Using-the-master-playback-rate-to-set-tempo-ae4e9a2813c4471e9aa658753702f878
• https://reapertips.notion.site/A-tool-for-customizing-REAPER-6-theme-cb0f56b6889441e8bb9b2e3d3b14a88c

I noticed that one thing that all of these have in common is that they have Youtube videos embed. So I deleted all youtube embeds and reuploaded and all body data outputs fine. So I think it has something to do with the Youtube embed that makes data body data go "{}"

What's weird, though, is that I've uploaded a lot of other articles with youtube videos successfully.

Here's some articles that have Youtube embed yet still get exported fine:
• https://reapertips.notion.site/Easily-create-stems-with-Stem-Manager-1da03bfbf4c14b2ab19e213c989a8462
• https://reapertips.notion.site/JS-Ravager-The-extreme-upward-compressor-b0cd3aab7b4c4ae8824e6845b667b81b

**Comment by alez156** [comment]
I was able to spot what I think might be the reason! I made a bunch of article variations to spot the problem. Here are the results: 
![Screen Shot 2022-11-15 at 7 54 31 PM](https://user-images.githubusercontent.com/21991578/202049473-7af51dbd-8712-4611-b22c-cb49e4c52615.png)

The error happens when an article has both images and videos. 
If it's only videos it will export successfully. But when the article has video somewhere and an image, then the output becomes nothing "{}"
